pootle/apps/pootle_translationproject/models.py

```diff
@@ -258,8 +271,21 @@ def save(self, *args, **kwargs):
         super(TranslationProject, self).save(*args, **kwargs)
 
         if created:
+            if initialize_from_templates:
+                # We are adding a new TP and there are no files to import from
+                # disk, so initialize the TP files using the templates TP ones.
+                from pootle_app.project_tree import init_store_from_template
+
+                for template_store in template_tp.stores.iterator():
+                    init_store_from_template(self, template_store)
+
             self.scan_files()
 
+            if initialize_from_templates:
```

Comment:
Let's add a comment here indicating that this won't be necessary once the work for #3547 has been completed (more specifically [this will supersede it](https://github.com/translate/pootle/pull/3583/files#diff-f684a8a253f640ccaace73ac502ad438R68)).